Grammar improvements and simplifications in the Getting Started Section

pavlenex opened the pull request:
Had some time to re-read our Getting Started section and did some changes in the following pages:
 - `/introduction`
 - `/why-bitcoin-is-unique`
 - `why-design-for-bitcoin`

Happy to continue through the section if this kind of fixes are accepted.

GBKS commented:
Thank you for this review. Super helpful and makes me feel more comfortable knowing that the content has undergone some serious scrutiny. Whenever you're ready, let me know and I'll review all your changes and approve.

pavlenex commented:
@GBKS over to you now!

GBKS commented on guide/getting-started/getting-started.md:
```diff
@@ -34,61 +34,61 @@ Illustration sources
 # Getting started
 
-Welcome to bitcoin design, you're in for an adventure. This chapter is all about the big concepts that are helpful to be familiar with when creating bitcoin products. You'll find a brief summary of each concept on this page, click through if you would like to dig deeper into a specific topic. Note that this is not meant as a comprehensive overview of all topics bitcoin, but only the ones we think ...
+Welcome to Bitcoin Design Guide Guide! You’re in for an adventure. This chapter is all about the big concepts that are helpful to be familiar with when creating Bitcoin products. You’ll find a summary of concepts on this page, click through if you would like to dig deeper into a specific topic. The guide isn't a comprehensive overview of all Bitcoin topics, but only the ones you should know as a  ...
```
```suggestion
Welcome to the Bitcoin Design Guide! You’re in for an adventure. This chapter is all about the big concepts that are helpful to be familiar with when creating Bitcoin products. You’ll find a summary of concepts on this page, click through if you would like to dig deeper into a specific topic. The guide isn't a comprehensive overview of all Bitcoin topics, but only the ones you should know as a designer working in this space.
```

GBKS commented on guide/getting-started/getting-started.md:
```diff
@@ -34,61 +34,61 @@ Illustration sources
 **[Open design]({{ '/guide/getting-started/open-design/' | relative_url }})**
 
-Open design is about global collaboration that balances the free exchange of design information, the independence of individual designers and the collective power of collaboration.
+Open design is about global collaboration that balances the free exchange of design information, individual designers' independence, and the collective power of collaboration.
```
I prefer the original phrasing, it's simpler to read.

pavlenex replied:
You're right, reverted in https://github.com/BitcoinDesign/Guide/commit/8796246fd2cb09bb9065848f6f28145b3566637d

GBKS commented on guide/getting-started/getting-started.md:
```diff
@@ -34,61 +34,61 @@ Illustration sources
 **[Common user flows]({{ '/guide/getting-started/common-user-flows/' | relative_url }})**
 
-Most bitcoin applications share certain user activities, from initial setup to making a transaction. It is important to follow expected patterns and behaviors users to reduce the risk of mistakes and allow users to seamlessly move between products.
+Most Bitcoin applications share certain user activities, from initial setup to making a transaction. It is vital to follow users' expected patterns and behaviors to reduce mistakes and enable seamless move between products.
```
```suggestion
Most Bitcoin applications share certain user activities, from initial setup to making a transaction. It is vital to follow users' expected patterns and behaviors to reduce mistakes and enable seamless switching between products.
```

GBKS commented on guide/getting-started/open-design.md:
```diff
@@ -52,63 +52,65 @@ Illustration sources
+- are open-minded and flexible
+- work multidisciplinary because good results require different perspectives and skillsets
+- continuously learn independently and from each other
+- rely on our peer group for free and communal validation of a quality
```
```suggestion
- rely on our peer group for free and communal validation of quality
```

GBKS commented on guide/getting-started/common-user-flows.md:
```diff
@@ -166,7 +166,7 @@ One of the most complex users flows in Bitcoin is the creation of multisignature
 %}
 
-We ask users to securely back up their keys when they create new wallets so they can restore them later. This might be due to practical reasons like switching to a new computer or from a laptop to a smartphone. It may also be part of an investment strategy, like when a user wants to access bitcoin from cold storage after not interacting with them for months or years.
+We ask users to securely back up their keys when they create new wallets to restore them later. This might be due to practical reasons like switching to a new computer or from a laptop to a smartphone. It may also be part of an investment strategy, like when a user wants to access bitcoin from cold storage after not interacting with them for months or years.
```
```suggestion
We ask users to securely back up their keys when they create new wallets, to restore them later. This might be due to practical reasons like switching to a new computer or from a laptop to a smartphone. It may also be part of an investment strategy, like when a user wants to access bitcoin from cold storage after not interacting with them for months or years.
```
Might need this extra comma so the sentence doesn't get confusing (restoring is connected to backing up keys, not to creating wallets).

GBKS commented on guide/getting-started/common-user-flows.md:
```diff
@@ -241,11 +240,11 @@ Equivalent to creating an invoice, requesting bitcoin involves entering informat
 %}
 
-While it is initially intuitive to re-use the same receiving address over and over, this practice is highly discouraged. Anyone with this address can easily see what other payments are being made to it and how the bitcoin are spent. The UI should encourage users to create new addresses for each invoice. This also makes it easier for the software to understand when a specific payment has been rece ...
+While it is initially intuitive to re-use the same receiving address repeatedly, this practice is highly discouraged. Anyone with this address can easily see what other payments are being made and how the bitcoin is spent. The UI should encourage users to create new addresses for each invoice. This also makes it easier for the software to understand when a specific payment has been received.
```
> how the bitcoin is spent

Shouldn't it be plural as it's a quantity? "How the Euro is spent" also doesn't sound right.

pavlenex replied:
Fixed in https://github.com/BitcoinDesign/Guide/commit/8796246fd2cb09bb9065848f6f28145b3566637d

GBKS commented on guide/getting-started/personal-finance.md:
```diff
@@ -36,13 +36,13 @@ Illustration sources
-Money is complex and can play many different roles in our lives. Buying a new shirt is a very different transaction than buying a house or setting aside savings for retirement, and taking a payment as a store owner is very different from a child receiving allowance from parents. On a global scale, the differences multiply across countries and cultures.
+Money is complex and can play many different roles in our lives. Buying a new shirt is quite different from buying a house or setting aside savings for retirement. Taking a payment as a store owner is very different from a child receiving an allowance from parents. On a global scale, the differences multiply across countries and cultures.
 
-It still is possible to create simplified mental models that help us better understand user needs and behaviors. A single model might be oversimplified, but if we combine a few different lenses we can narrow in on a more complete picture that allows us to create more focused experiences, have better product differentiation, and make it easier for contributors to collaborate to make decisions.
```
> A single model might be oversimplified, but if we combine a few different lenses we can narrow in on a more complete picture that allows us to create more focused experiences, have better product differentiation, and make it easier for contributors to collaborate to make decisions.

Why remove?

pavlenex replied:
Quite honestly, it sounded complicated, in the hindsight. We should tackle removals in a different or,

Reverted https://github.com/BitcoinDesign/Guide/pull/170/commits/8796246fd2cb09bb9065848f6f28145b3566637d

GBKS commented on guide/getting-started/software.md:
```diff
@@ -84,17 +84,14 @@ Explorers often also provide insight into activity on the bitcoin network, like
 
-Wallet provide easy-to-use graphical interfaces for a wide audience. Wallets many times don't offer all functionality that can be found in a CLI, but instead focus on general use cases like making payments and browsing transactions.
-
-Due to the nature of open-source, wallets can be developed by anyone with the technical knowledge, and many code libraries are available to simplify this task.
```
> Due to the nature of open-source, wallets can be developed by anyone with the technical knowledge, and many code libraries are available to simplify this task.

Why remove?

pavlenex replied:
Reverted 8796246

GBKS commented on guide/getting-started/why-design-for-bitcoin.md:
```diff
@@ -37,38 +37,36 @@ Illustration sources
 ## Design for your needs
 
-Many bitcoin projects start by identifying an unmet need. Unlike traditional finance, the barriers to entry are incredibly low. So if you were ever unhappy with the service of your bank for financial provider, here's your chance to shape something better.
+Many bitcoin projects start by identifying an unmet need. Unlike traditional finance, the barriers to entry are shallow. So if you were ever unhappy with your bank's service of a financial provider, here's your chance to shape something better.
```
```suggestion
Many bitcoin projects start by identifying an unmet need. Unlike traditional finance, the barriers to entry are shallow. So if you were ever unhappy with the service of your bank or financial provider, here's your chance to shape something better.
```
The fix was not quite right due to my initial mistake of writing "for" instead of "or".

GBKS commented:
@pavlenex thanks for that super thorough review. I went over everything and commented when necessary. Did you notice any larger gaps or problems with the content?

pavlenex commented:
@GBKS Thanks for reviewing. I reverted the removal of two missing paragraphs in https://github.com/BitcoinDesign/Guide/commit/8796246fd2cb09bb9065848f6f28145b3566637d and addressed two other comments you had.

GBKS reviewed (approved):
Super awesome PR with a huge amount of word polish.
